src/rcore.c

```diff
@@ -2957,13 +2957,22 @@ int GetGamepadAxisCount(int gamepad)
     return CORE.Input.Gamepad.axisCount[gamepad];
 }
 
+static float GetGamepadAxisMovementDefault(int axis)
```

Comment:
I prefer to avoid adding new functions. Please, could you use that line directly on target function?

Reply:
Sure, no problem. Inlined it.